test/input-excheptions.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh -e
```

Comment:
How come this test exists both as input-excheptions.sh and t3990-input-exceptions.sh?